packages/odyssey-react/src/components/Table/TableSortButton.tsx

```diff
@@ -41,13 +76,20 @@ const TableSortButton = forwardRef<Ref, Props>((props, ref) => {
     <button ref={ref} className={componentClass} {...omitProps}>
       {children}
       <span className={styles.sortIndicator}>
-        {direction === "unsorted" && <SortIcon title="Unsorted" />}
-        {direction === "asc" && <SortAscIcon title="Ascending" />}
-        {direction === "desc" && <SortDescIcon title="Descending" />}
+        {direction === "unsorted" && <SortIcon title={unsortedIconTitle} />}
+        {direction === "asc" && <SortAscIcon title={ascendingIconTitle} />}
+        {direction === "desc" && <SortDescIcon title={descendingIconTitle} />}
       </span>
-      <ScreenReaderText>click to sort</ScreenReaderText>
+      <ScreenReaderText>{screenReaderCallToAction}</ScreenReaderText>
     </button>
   );
 });
 
+TableSortButton.displayName = "TableSortButton";
+
+TableSortButton = withStyles(styles)(TableSortButton);
```

Comment:
as we discussed in office hours previously - these statics based APIs that are all sharing a single style module are going to be introducing some implicit dependencies on the parent being loaded. It also means that it is strictly unnecessary for us to wrap `TableSortButton` in withStyles here since they are referencing the same module `Table.module.scss`. In the near future we should do a pass of all components sharing a style module and make an effort to separate them - there is no reason for this implicit hard to follow API if each component can simply work from their own style sheet to increase portability and composition based APIs for customers